test/functional-api/user-management/application/application.it-spec.ts

```diff
@@ -123,12 +192,178 @@ describe('Application', () => {
 
     // Act
     let removeApp = await removeApplicationMutation(applicationId);
-    const getApp = await getApplication(applicationId);
+    const getApp = await getApplication(ecoverseId, applicationId);
 
     // Assert
     expect(removeApp.status).toBe(200);
     expect(getApp.text).toContain(
       `Application with ID ${applicationId} can not be found!`
     );
   });
+
+  // Bug - user can create challenge application, when there is no ecoverse application
```

Comment:
Let's address that on the board and put the ID in this comment for proper tracking.